exercise/change_attributes.py

```diff
@@ -0,0 +1,147 @@
+"""
+This module contains a code for exercises 17-1 related to:
+Think Python, 2nd Edition
+Chapter 17: Classes and Methods
+
+Modify the methods (and the function int_to_time) to work with the
+new implementation.
+"""
+
+class Time(object):
+    """
+    Represents the time of day.
+    """
+
+    def __init__(self, hour=0, minute=0, second=0):
+        """
+        Initializes a Time object.
+
+        Attributes: hour, minute, second
+        """
+        
+        minutes = hour * 60 + minute
+        self.int_time = (minutes * 60) + second
+
+    def __str__(self):
+        """
+        Returns a string representation of the Time object in the form 'hour:minute:second'.
+
+        Returns:
+            str: A formatted string representing the time.
+        """
+        
+        minutes, second = divmod(self.int_time, 60)
+        hour, minute = divmod(minutes, 60)
+        return "%.2d:%.2d:%.2d" % (hour, minute, second)
```

Comment:
Should add the empty line before the return statement.
Double-check for all files in this PR